src/js/component/Home.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import List from "./List.jsx";
+
+const Home = () => {
+	return (
+		<>
+			<List placeholder="List of To Do..."></List>
+		</>
+	);
+};
+
+export default Home;
```

Comment:
I think you have not listened my requirements. Please try to explain me what is the main functionality of this component `Home`?
I know you don't know what are you doing, this software layer called `Home` gives to your code more complexity and it's not exactly what I want. 
You must change it and adapt your code to acomplish this issue.

Reply:
I'm deleting the component Home, unnecesary component in the global functionality of the ToDoList